schedule/cryptlvm_minimal_x.yaml

```diff
@@ -0,0 +1,60 @@
+---
+name:           cryptlvm_minimal_x
+description:    >
+  Combination of "cryptlvm" and "minimal_x" for sle12.
+  (crypt-)LVM installations can take longer,
+  especially on non-x86_64 architectures.
+vars:
+  ENCRYPT: 1  
+  LVM: 1
+  MAX_JOB_TIME: 14400
+schedule:
+  - installation/bootloader
+  - installation/welcome
+  - installation/accept_license
+  - installation/registration
```

Comment:
rename back to [scc_registration](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/installation/scc_registration.pm)

Reply:
done